locale-utils.js

```diff
@@ -78,6 +78,29 @@ const LocaleUtils = {
     }
     return id;
   },
+
+  fluentFormatWithFallback (supportedLocales, id, fallbackId, args=null, errors=null) {
```

Comment:
This function will see if the new string is translated. If not, it will fallback to the previous string that is already translated. 

cc/ @flodolo @tcinotto

Reply:
Is this preferable to falling back to the next best language?

Reply:
Yes because the previous/fallback strings in question are already translated. (For German and French). This is a convention used on [MozOrg/Bedrock](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/includes/protocol/footer/footer.html#L23). We didn't get the updated strings until Friday afternoon, and the original plan was to revert to the previous strings.